### PR TITLE
read-through of paper

### PR DESCRIPTION
**Pull request by mhagdorn**
Some minor changes I noticed re-reading the paper in its entirety.

### CODE REVIEW

File: competencies.md [hunk 1]
```diff
@@ -305,7 +305,7 @@ Through writing research software, RSEs have a pivotal position in the process o
 Their choices might determine whether the respective research is reproducible or not,
 whether the results can be re-used, whether future research can build on existing tools or has to start from scratch.
 Builders of larger research-infrastructure projects determine to some extent the possibilities and limitations of future research
-and therefore need to be able to make a value-based judgement on topicshttps://hpc.fau.de/systems-services/documentation-instructions/nhr-application-rules/
```
Comment (mhagdorn) [inline competencies.md:308]: presumably this URL got pasted in by accident

File: competencies.md [hunk 2]
```diff
@@ -465,8 +465,6 @@ warrants a much stronger emphasis on this field of competence.
 Working in a team is all about communication and teamwork.
 For example, RSEs need to be able to explain particular implementation choices made and may even need to defend them.
 Within a team of RSEs code reviews improve knowledge transfer and increase team cohesion.
-Code review within a multi-disciplinary team fosters mutual understanding and
```
Comment (mhagdorn) [inline competencies.md:468]: repeat of previous sentence

File: competencies.md [hunk 3]
```diff
@@ -963,7 +961,7 @@ This can often serve as a base domain for RSE specialisation as in @subsec:examp
 #### Data-focused RSE
 RSEs working at the flourishing intersection between data science and RSE.
 They are skilled in cleaning data and/or running data analyses and can help researchers
-in setting up their analysis pipeline and/or research data management (RDM) solutions.
```
Comment (CaptainSifff) [inline competencies.md:966]: RDM is first used in the OS-RSE, so we should move that over there.

**Review by CaptainSifff — CHANGES_REQUESTED** [review]

File: competencies.md (hunk 3 above)
Comment (CaptainSifff) [reply competencies.md:966]: sorry, found it....

**Pull request merged**